src/execution-metrics.ts

```diff
@@ -14,6 +14,7 @@ export class ExecutionMetrics {
     public skippedSpecs = 0;
     public totalSpecsDefined = 0;
     public executedSpecs = 0;
+    public errors: CustomReporterResult[] = [];
```

Comment:
In jasmine default reporter, this kind of error is specific to afterAll error, what about naming it `afterAllErrors` instead of `errors`?
Then, the summary report should also be something like `* AfterAll Errors *`